<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>تکوینگر</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidgetTopArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="page_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCommandLinkButton" name="commandLinkButton_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>CommandLinkButton</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCommandLinkButton" name="commandLinkButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>CommandLinkButton</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCommandLinkButton" name="commandLinkButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>CommandLinkButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3"/>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidgetMain">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageMdiArea">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QMdiArea" name="mdiArea"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>فایل</string>
    </property>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>ابزارها</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="separator"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>راهنما</string>
    </property>
    <addaction name="actHelp"/>
    <addaction name="actAbout"/>
   </widget>
   <widget class="QMenu" name="menulndvdj">
    <property name="title">
     <string>مدیریت</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menulndvdj"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actHelp">
   <property name="text">
    <string>راهنمای برنامه</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>درباره برنامه</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>بستن</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>محاسبه کارمزد</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>فاکتور نویس</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>پروژه‌ها</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>پرسنل</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>سهامداران</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
